iceoryx_posh/include/iceoryx_posh/internal/popo/building_blocks/chunk_receiver.hpp

```diff
@@ -26,7 +26,7 @@ namespace iox
 {
 namespace popo
 {
-enum class ChunkReceiverError
+enum class ChunkReceiveError
```

Comment:
why this change?

Reply:
I use the same error codes again for the subscriber. So it is no more the error for the chunk receiver but more general an error that can occur when a chunk is received, either with ChunkReceiver or with SubscriberPort